common/bin-common/src/lib.rs

```diff
@@ -10,3 +10,6 @@ pub mod completions;
 
 #[cfg(feature = "output_format")]
 pub mod output_format;
+
+#[cfg(feature = "models")]
+pub mod models;
```

Comment:
> why would say nym-api or nym-client ever care about those models?

if they don't, that's why I put it behind `models`

I needed a place from which both `node-status-api` and `node-status-agent` could pull a depending struct because i'd like to avoid having one of them depend on the other. 

Can you show me a better place to put it?

Reply:
I'd say create another `common/<something>` crate, I don't want `bin-common` getting polluted because it's meant to be for common things shared by runnable binaries: like build information, setting up logging, etc.

Reply:
done, moved to a lib called `common/models`